ichnaea/scripts/importer.py

```diff
@@ -44,8 +43,9 @@ def load_file(session, source_file, batch_size=1000, userid=None):
                     # convert from unixtime to utc
                     time = datetime.datetime.utcfromtimestamp(time)
 
-                key = normalized_wifi_key(str(fields[1]))
-                if not valid_wifi_pattern(key):  # pragma: no cover
+                try:
+                    key = WifiKeyNode(String()).deserialize(str(fields[1]))
```

Comment:
I do think, however, that eventually we should call the colander validators directly instead of using the normalize_thing_dict interfaces.  I left them intact for now because we can worry about refactoring those bits later.